spec/index.html

```diff
@@ -889,6 +913,9 @@ <h3>Triple Terms</h3>
 
     <p>By extension, an [=RDF term=] is said to [=appear=] in an [=RDF graph=] if it appears in an [=asserted triple=] of that graph. An [=RDF triple=] is said to [=appear=] in an [=RDF graph=] if it is either an [=asserted triple=] of that graph or a [=triple term=] [=appearing=] in that graph.</p>
 
+    <p><dfn>Triple term equality</dfn>:
+      Since triple terms are [=triples=], equality of triple terms is the same as [=triple equality=].</p>
+
```

Comment:
```suggestion
    <p>Triple term equality:
      Since triple terms are [=triples=], equality of triple terms is the same as [=triple equality=].</p>
      
```
For the sake of regularity, I understand that it is desirable to have a paragraph about "Triple term equality". But I think that making it a definition is not required and adds confusion.  As this paragraph explains, triples terms *are* triples, so triple equality applies here.